html/canvas/element/manual/imagebitmap/createImageBitmap-bounds.html

```diff
@@ -4,41 +4,60 @@
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script src="/html/canvas/resources/canvas-tests.js"></script>
+<div id="results"></div>
 <script>
-promise_test(function(t) {
-    return new Promise(function(resolve, reject) {
-        const image = new Image();
-        image.onload = function() { resolve(image); };
-        image.onerror = function() { reject(); };
-        image.src = "/images/green-16x16.png";
-    }).then(function(image) {
-        return createImageBitmap(image, 8, 8, 16, 16);
-    }).then(function(imageBitmap) {
-        const color = 204;
+const color = 204;
+function testClip( name, sx, sy, sw, sh, expected ) {
+    promise_test(function(t) {
+        return new Promise(function(resolve, reject) {
+            const image = new Image();
+            image.onload = function() { resolve(image); };
+            image.onerror = function() { reject(); };
+            image.src = "/images/green-16x16.png";
+        }).then(function(image) {
+            return createImageBitmap(image, sx, sy, sw, sh);
+        }).then(function(imageBitmap) {
+            const canvas = document.createElement("canvas");
+            canvas.width = 16;
+            canvas.height = 16;
 
-        const canvas = document.createElement("canvas");
-        canvas.width = 16;
-        canvas.height = 16;
+            // debug
```

Comment:
Is this useful for those viewing the test? If so, maybe make that more explicit.

Reply:
That was actually there in the previous version.
I only made the canvases all fit in the same container (otherwise new ones would have been after test results).
Not sure how useful it is, nor how to make that more explicit, any suggestion?

Reply:
Maybe "Show canvas for those running the tests manually." but feel free to disregard given it was an existing issue.